src/config.ts

```diff
@@ -180,4 +180,4 @@ export const BLOCKCHAIN_STORE_PATH = path.join(
   electronStore.get("BlockchainStoreDirName")
 );
 export const MIXPANEL_TOKEN = "80a1e14b57d050536185c7459d45195a";
-export const TRANSIFEX_TOKEN = "1/1ddc990ff2fdce9ab9cc5b8c38af80c2dfe3d358"; // TODO: Change this
+export const TRANSIFEX_TOKEN = "1/a0e9b601e795f5839d6f78e29fcf455175000672"; // TODO: Change this
```

Comment:
Do we still need to change it?
```suggestion
export const TRANSIFEX_TOKEN = "1/a0e9b601e795f5839d6f78e29fcf455175000672";
```

Reply:
Oops, totally missed it.